app/views/shared/_validated_input.html.haml

```diff
@@ -1,4 +1,4 @@
-%label{for: path}= name
+%label{for: path, "ng-class": "{error: !fieldValid('#{path}')}" }= name
```

Comment:
We should drop this change. We're not using this any more and I think it is used elsewhere in the app.

Reply:
Right. I dropped the commit.